### ISSUE TITLE
[bootstrap] /usr/bin/python: not found

### ISSUE THREAD

**Issue by nqb** [debops/debops#1002]
As mentioned on IRC, if you try to bootstrap a host with ansible < 2.8, the first play of the bootstrap playbook will uninstall Python 2 environment on host causing next plays to fail with following error:

```console
 fatal: [debops-a3bc65]: FAILED! => {"changed": false, "module_stderr": "/bin/sh: 1: /usr/bin/python: not found\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 127}

```
Some useful informations:
- https://docs.ansible.com/ansible/2.8/porting_guides/porting_guide_2.8.html#python-interpreter-discovery
- https://docs.ansible.com/ansible/2.8/reference_appendices/interpreter_discovery.html

Default configuration for an ansible 2.8:
```console
$ ansible-config dump | grep -i interpreter
INTERPRETER_PYTHON(default) = auto_legacy
INTERPRETER_PYTHON_DISTRO_MAP(default) = {'centos': {'6': '/usr/bin/python', '8': '/usr/libexec/platform-python'}, 'fedora': {'23': '/usr/bin/python3'}, 'redhat': {'6': '/usr/bin/python', '8': '/usr/libexec/platform-python'}, 'rhel': {'6': '/usr/bin/python', '8': '/usr/libexec/platform-python'}, 'ubuntu': {'14': '/usr/bin/python', '16': '/usr/bin/python3'}}
INTERPRETER_PYTHON_FALLBACK(default) = ['/usr/bin/python', 'python3.7', 'python3.6', 'python3.5', 'python2.7', 'python2.6', '/usr/libexec/platform-python', '/usr/bin/python3', 'python']
```
=> Debian is not part of `INTERPRETER_PYTHON_DISTRO_MAP`

### Steps to reproduce
Try to bootstrap a host with bootstrap playbook using ansible < 2.8

### Actual results
ansible is unable to detect Python 3 interpreter.

### Expected results
ansible should find Python 3 interpreter.

### Solution
Because we need to support ansible < 2.8 for a while:
- Stretch comes with ansible 2.7.5 (backports)
- Buster comes with ansible 2.7.7 (stable)

and even with ansible 2.8, Debian python interpreter came from fallback list, it seems better for now to set `ansible_python_interpreter` to `/usr/bin/python3` maybe using `debops-init`.

### Workarounds
- Set `python__v2: True` in your Ansible inventory (to avoid removing Python 2 interpreter)
- Set `ansible_python_interpreter=/usr/bin/python3` in your Ansible inventory

**Comment by drybjed** [comment]
I agree that `debops-init` should conigure the interpreter in the generated inventory by default, perhaps by creating the `ansible/inventory/group_vars/all/ansible_meta.yml` file with `ansible_python_interpreter` variable. Going back to Python 2.7 seems pointless at this point, with it being end of life in a few months.

One wrinkle in this plan is that Debian Stretch does not provide `python3-ldap` package, but it can be worked around by installing the `python-ldap` Python package from PyPI, which is what the default installation instructions do. And, since Stretch is currently Oldstable and new OS installations will most likely use Buster, it will not be a problem for much longer, I hope.
